tests/conftest.py

```diff
@@ -116,21 +170,21 @@ def config_str(gordo_name: str, sensors: List[SensorTag]):
                         - sklearn.preprocessing.data.MinMaxScaler
                         - gordo_components.machine.model.models.KerasAutoEncoder:
                             kind: feedforward_hourglass
-                name: {gordo_name}
+                name: {gordo_single_target}
              """
 
 
 @pytest.fixture(scope="session")
 def trained_model_directory(
-    model_collection_directory: str, config_str: str, gordo_name: str
+    model_collection_directory: str, config_str: str, gordo_single_target: str
 ):
     """
     Fixture: Train a basic AutoEncoder and save it to a given directory
     will also save some metadata with the model
     """
 
     # Model specific to the model being trained here
-    model_dir = os.path.join(model_collection_directory, gordo_name)
+    model_dir = os.path.join(model_collection_directory, gordo_single_target)
```

Comment:
One thing worth mentioning here... `gordo_single_target` is just the gordo name, but when used in the client/server tests makes sense contextually to be called `gordo_single_target`, but in other places this feels to me to be slightly ambiguous. 

I decided to just use `gordo_single_target` throughout, but perhaps we want to change it to `gordo_name`.

Reply:
Yeah, I found this name change a bit confusing maybe, for the other (non client/server) places.

Reply:
Do you think I should maybe alias `gordo_name` to `gordo_single_target`, or swap out `gordo_single_target` for `gordo_name?

Reply:
I've now added the `gordo_name` fixture and an alias to a `gordo_single_target` fixture :+1: